### PR TITLE
Add Fix for Mafia II: Definitive Edition

### PR DESCRIPTION
**Pull request by springwurm**
See Proton Issue https://github.com/ValveSoftware/Proton/issues/3891

### CODE REVIEW

File: protonfixes/gamefixes/1030830.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from protonfixes import util
+from protonfixes.logger import log
+
+def main():
+    """ Changes the proton argument from the launcher to the game
+    """
+
+    log('Applying fixes for Mafia II: Definitive Edition')
+
+    util.protontricks('dotnet542')
```
Comment (t1764722) [inline protonfixes/gamefixes/1030830.py:14]: util.protontricks('dotnet452')

**Comment by t1764722** [comment]
I think there's a typo line 14. should be dotnet452 not dotnet542.

**Comment by springwurm** [comment]
> I think there's a typo line 14. should be dotnet452 not dotnet542.

actually, i thing dotnet is only for the Luncher.
I removed the protontricks line and the "Mafia II: Definitive Edition" Proton Prefix directory.

The Game worked regardless.

**Comment by kernle32dll** [comment]
I recon the same fix can be applied for `Mafia III: Definitive Edition`?

**Pull request merged**